.changeset/mighty-steaks-lie.md

```diff
@@ -0,0 +1,5 @@
+---
+"@preact/signals-core": patch
+---
+
+Fix botched publish
```

Comment:
`@preact/signals-core v1.3.0` accidentally includes a tarball of v1.2.2 cuz I forgot to clean before publishing. So creating a dummy patch version so I can republish